code/world.dm

```diff
@@ -12,6 +12,16 @@
 
 var/savefile/panicfile
 /world/New()
+	if(fexists("byond-extools.dll") && system_type == MS_WINDOWS)
+		call("byond-extools.dll", "maptick_initialize")()
+	else if(system_type == UNIX)
```

Comment:
The path to the dll should be the conditional here.
```dm
var/extools_path = system_type == MS_WINDOWS ? "byond-extools.dll" : "libbyond-extools.so"
```

Reply:
Functionally identical and not really important, is it?